examples/2016-10-31/usage_plan/template.yaml

```diff
@@ -0,0 +1,96 @@
+Conditions:
+  C1:
+    Fn::Equals:
+      - true
+      - true
+
+Parameters:
+  UsagePlanType:
+    Type: String
+    Default: PER_API
+
+Globals:
+    Api:
+      OpenApiVersion: 3.0.0
+      Auth:
+        ApiKeyRequired: true
+        UsagePlan:
+          CreateUsagePlan: !Ref UsagePlanType
+
+Resources:
+   MyApiOne:
+     Condition: C1
+     Type: AWS::Serverless::Api
+     Properties:
+       StageName: Prod
```

Comment:
Might be out of scope for this example, since this doesn't use a usage plan at all.